multivariable-calculus/manipulating-derivatives.md

```diff
@@ -1 +1,67 @@
-# Manipulating partial derivatives
+# Manipulating Partial Derivatives
+
+## Example:
```

Comment:
We can wrap all this in an "example" environment so it shows up like the example here:
https://mphowardlab.github.io/essential-math/linear-algebra/matrices.html

```suggestion
````{example} Change in internal energy with temperature and pressure
```

You will need to put quadruple ticks at the end of the example to close this block. You can check `linear-algebra/matrices.md` for an example.